cmd/snap-seccomp/versioninfo.go

```diff
@@ -0,0 +1,503 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"os"
+
+	"github.com/mvo5/libseccomp-golang"
+
+	"github.com/snapcore/snapd/osutil"
+)
+
+// Generated using arch-syscall-dump test tool from libseccomp tree, git
+// revision 584ca7a5e69d87a4c2c4e4c07ce8415fa59e1351.
+var seccompSyscalls = []string{
```

Comment:
Would you mid if I follow up by moving this to `sandbox/syscalls` or `sandbox/seccomp/syscalls`?

Reply:
first we need to agree what this list is and how we change it, before discussing how to move it

Reply:
I think it's fine to have a complete list of system calls as long as we also ensure that we correctly use only a subset of those.